spec/dealership_spec.rb

```diff
@@ -0,0 +1,74 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+
```

Comment:
I would add an `it exisits` test to test that a `dealership` is and instance of `Dealership`.